code/modules/mob/living/carbon/human/human.dm

```diff
@@ -650,46 +650,71 @@
 				to_chat(src, "<span class='warning'>\The [S] pulls \the [hand] from your grip!</span>")
 	rad_act(current_size * 3)
 
-/mob/living/carbon/human/proc/do_cpr(mob/living/carbon/C)
+#define CPR_PANIC_SPEED (0.8 SECONDS)
+
+/// Performs CPR on the target after a delay.
+/mob/living/carbon/human/proc/do_cpr(mob/living/carbon/C, panicking)
```

Comment:
Could you change `C` to a better variable name, such as `target`, `carbon_target`, `cpr_target` or the likes?